.github/workflows/build.yml

```diff
@@ -7,43 +7,134 @@ name: Build
 
 on:
   push:
-    branches: [main]
+    branches:
+      - main
   pull_request:
-    branches: [main]
-  workflow_dispatch: null
+    branches:
+      - main
+  schedule:
+    - cron: 0 0 * * 1
 
 jobs:
-  build:
-    name: Build
+  deploy:
+    name: Deploy
     runs-on: ubuntu-latest
+    permissions: {}
     steps:
       - name: Checkout
         uses: actions/checkout@v3
 
+      - name: npm – Install
+        shell: pwsh
+        run: npm install
+
+      - name:
+        shell: pwsh
+        # Azure DevOps Personal Access Token (PAT) with the Agent Pools (Read & manage) permission.
+        run: npx tfx login --service-url https://${{ secrets.AZDO_ACCOUNT }}.visualstudio.com/DefaultCollection --token ${{ secrets.AZDO_PAT }}
+
+      - name: npm – Deploy
+        shell: pwsh
+        run: npm run deploy
+
+  update-code:
+    name: Update Code
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          ref: ${{ github.HEAD_REF }}
+          token: ${{ secrets.BUILD_UPDATE_CODE_CHECKOUT }}
+
       - name: Install Node.js
         uses: actions/setup-node@v3
         with:
           node-version: 19.x
 
-      - name: NPM – Install Dependencies
+      - name: npm – Install Dependencies
+        shell: pwsh
         run: npm install
 
-      - name: NPM – Test Lint
+      - name: npm – Lint
+        shell: pwsh
         run: npm run lint
 
-      - name: NPM – Test Clean
+      - name: npm – Build Package
+        shell: pwsh
+        run: npm run build:package
+
+      - name: Git – Add Changed Files
+        shell: pwsh
+        run: git add -A
+
+      - name: Detect Changes
+        id: detect-changes
+        shell: pwsh
+        run: |-
+          $GitStatus = git status
+          Write-Output -InputObject $GitStatus
+          $NoChangesPresent = $GitStatus.Contains("nothing to commit, working tree clean")
+          Write-Output -InputObject $NoChangesPresent
+          Write-Output -InputObject "NO_CHANGES_PRESENT=$NoChangesPresent" >> $Env:GITHUB_OUTPUT
+
+      - if: ${{ steps.detect-changes.outputs.NO_CHANGES_PRESENT == 'False' }}
+        name: Git – Set Name
+        shell: pwsh
+        run: git config --global user.name "github-actions[bot]"
+
+      - if: ${{ steps.detect-changes.outputs.NO_CHANGES_PRESENT == 'False' }}
+        name: Git – Set Email
+        shell: pwsh
+        run: git config --global user.email "41898282+github-actions[bot]@users.noreply.github.com"
```

Comment:
Is there any concern with adding this email account here?

Reply:
It's the GitHub Bot, and the details are well known, see https://github.com/orgs/community/discussions/26560 for example. I don't believe there should be a problem with this.